app/models/etymology.rb

```diff
@@ -0,0 +1,5 @@
+class Etymology < ApplicationRecord
+  belongs_to :context, polymorphic: true
+
+  has_many :etymology_subject_associations, dependent: :destroy
```

Comment:
Would also add to restrict etymology type for specific etymology:
`has_one :etymology_type_association`
once EtymologyTypeAssociation (see below) has been created.